packages/workato-adapter/src/change_validators/actions_not_supported.ts

```diff
@@ -0,0 +1,41 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import {
+  ChangeError,
+  ChangeValidator,
+  getChangeData,
+  isInstanceChange,
+  isRemovalChange,
+} from '@salto-io/adapter-api'
+import { isFromType } from '../utils'
+import { REMOVAL_SUPPORTED_TYPES } from '../constants'
+
+export const changeValidator: ChangeValidator = async changes => (
+  changes
+    .filter(isInstanceChange)
+    .filter(isRemovalChange)
+    .map(getChangeData)
+    .filter(elem => !(isFromType(REMOVAL_SUPPORTED_TYPES)(elem)))
+    .map(element => ({
+      elemID: element.elemID,
+      severity: 'Error',
+      message: `Removing element of type '${element.elemID.typeName}' is not supported`,
+      detailedMessage: `Removing element of type '${element.elemID.typeName}' is not supported`,
+    } as ChangeError))
```

Comment:
I don't think we need the cast (here and in the other places)
```suggestion
    }))
```